priv/python/lux/prism.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+import uuid
+import os
+
+class Prism(ABC):
+    def __init__(self, **kwargs):
+        self.id = kwargs.get('id', str(uuid.uuid4()))
+        self.name = kwargs.get('name', '')
+        self.description = kwargs.get('description', '')
+        self.input_schema = kwargs.get('input_schema', {})
+        self.output_schema = kwargs.get('output_schema', {})
+    
+    def get_metadata(self):
```

Comment:
Prisms have the `view/0` function in elixir, which will return the Prism struct. Could we stick to that? We could return a python Dict., that will be translated to a Prism Struct automatic if returned back to elixir.